dspot/src/test/resources/test-projects/pom.xml

```diff
@@ -1,21 +1,36 @@
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
```

Comment:
Since you changed this `pom.xml`, to test the maven-plugin i assume, The test `fr.inria.diversify.mutant.descartes.DescartesTest#testInjectionOfDescartesIfNeeded` is failing, because it compared the content the `pom.xml` to a String. Could please update the [expected value](https://github.com/STAMP-project/dspot/blob/master/dspot/src/test/java/fr/inria/diversify/mutant/descartes/DescartesTest.java#L44) of this [test](https://github.com/STAMP-project/dspot/blob/master/dspot/src/test/java/fr/inria/diversify/mutant/descartes/DescartesTest.java#L24)